### ISSUE TITLE
Conflict resolution GUI fails to run on 0.19.0-beta.3

### ISSUE THREAD

**Issue by SplendidAlakey** [QuiltMC/quilt-loader#274]
**MC** 1.19.2
**QL** 0.19.0-beta.3

I'm able to consistently reproduce it by doing the following:

1. Install Indium, nothing else
2. Launch the game
3. Since Indium requires Sodium (and FAPI for that matter), the game crashes, but the Quilt's error GUI doesn't open
4. Java also fails to close gracefully, so we have to force kill it

Logs: https://gist.github.com/SplendidAlakey/db9d6841dc9fe2694fbfafb75a689eeb

**Comment by AlexIIL** [comment]
Okay this should be fixed in beta.5 - can you retry?

**Comment by SplendidAlakey** [comment]
Yep, fixed.

**Comment by SplendidAlakey** [comment]
Actually, no, hold on. The JDK actually still remains open, even though it says process closed. I found that out after I crashed and went to delete the test instance, but couldn't due to a hung up JDK process.

![image](https://user-images.githubusercontent.com/86702831/229341136-14371495-75e4-44e6-881f-14febe792dc1.png)

Logs: https://gist.github.com/SplendidAlakey/db9d6841dc9fe2694fbfafb75a689eeb
